doc/ref/chap-expr-lang.md

```diff
@@ -209,7 +209,32 @@ Note that these are distinct from `!  &&  ||`.
 
 ### ysh-arith
 
-    +  -  *  /   //   %   **
+YSH supports most of the arithmetic operators from Python. Notably, `%` differs
+from python as it represents remainder in YSH.
```

Comment:
Thanks for documenting this!

I would phrase this as - `/ and %` differ from Python since they round toward zero, not negative infinity.


https://www.oilshell.org/blog/2024/03/release-0.21.0.html#integers-dont-do-whatever-python-or-c-does

So BOTH `/` and `%` in YSH are different from Python, for negative numbers.  They follow C and JavaScript more.

I would indeed call our operation "remainder", although Python's could be called "remainder" too -- because they both go with division.  (Python chooses the divmod() name for some reason)